sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1337,7 +1337,7 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
 
   def duplicateMapKeyFoundError(key: Any): SparkRuntimeException = {
     new SparkRuntimeException(
-      errorClass = "_LEGACY_ERROR_TEMP_2127",
+      errorClass = "DUPLICATED_MAP_KEY",
```

Comment:
As you are here, please, change the quoting of config value to:
```scala
"lastWin" -> toSQLConfVal(SQLConf.MapKeyDedupPolicy.LAST_WIN.toString())))
```

Reply:
> Let's inline the config value LAST_WIN as we do for boolean configs, for instance:

So, let me just inline the "LAST_WIN" instead of quoting the config value.

Reply:
Ah, so do we want to use `toSQLConfVal` instead of `toSQLConf`?

Reply:
Updated!